app/transport/http/resp.go

```diff
@@ -0,0 +1,29 @@
+package http
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+//Usually, you need to make a rollback mechanism. If there is an error, roll back the changes
+//from the database. But we will not do that now. Therefore, we will not change the status code
+// after an error occurs
+func WriteJSONResponse(w http.ResponseWriter, statusCode int, data interface{}) {
```

Comment:
We would not use this function outside this package, a simple `respond`  name for it looks reasonable from my perspective, and of course it is up to you.

Reply:
Please fix `w` => `rw` because we are using ```rw http.ResponseWriter``` in `CreateUser`